bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -365,6 +382,31 @@ function findOutputApksHelper (dir, build_type, arch) {
     return ret;
 }
 
+function findOutputBundlesHelper (dir, build_type) {
+    var shellSilent = shell.config.silent;
+    shell.config.silent = true;
+
+    // list directory recursively
+    var ret = shell.ls('-R', dir).map(function (file) {
+        // ls does not include base directory
+        return path.join(dir, file);
+    }).filter(function (file) {
+        // find all bundles
+        return file.match(/\.aab?$/i);
+    }).filter(function (candidate) {
```

Comment:
I might rewrite this part as one big expression with a big ternary:
```js
    …).filter((candidate) => (
        // May need to choose between release and debug bundle:
        (build_type === 'debug')
            ? /debug/.exec(candidate)
            : (build_type === 'release')
                ? /release/.exec(candidate);
                : true;
    ));
```

or maybe use a switch statement for this?

I am writing ternaries like they do in Prettier, seems to be a new fad that I like. Or maybe it is not right here?

Reply:
I don't mind writing nested ternaries if that is the Cordova standard. If you're asking my personal opinion, I typically stay away from ternaries if you have more than 2 outcomes because I find explicit blocks easier to read.

Reply:
> I typically stay away from ternaries if you have more than 2 outcomes
> […]

Let’s hold off on the extended ternaries, then.

About the Cordova code base, a lot of it seems to have been originally written for less modern JavaScript over a half-decade ago. People have been modernizing the code base in several different ways, not sure how consistent it really is at this point.

I think it would be ideal for us to discuss some general coding style updates we should do in <https://github.com/apache/cordova/issues>, like we did for some other recent improvements, and work on applying these changes through the different components. All of these ideas take time, and I think none of us have any truly extra time these days.